journal-of-software.csl

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Journal of Software(Chinese)</title>
```

Comment:
why does this title have (chinese)? From what I see they publish in English. Also, the default-locale is set to en-US, so this wouldn't appear in chinese in the first place.